icu4c/source/common/unicode/platform.h

```diff
@@ -847,6 +847,14 @@ namespace std {
 #   define U_IMPORT 
 #endif
 
+#ifdef U_HIDDEN
```

Comment:
This is in a public API header, so it's public API unless we say otherwise.

It needs a comment block like the other definitions here (with a `\def U_HIDDEN` line to connect it with whatever is the active definition), and it should probably be marked as `@internal`.

Reply:
Made the above fix in https://github.com/unicode-org/icu/pull/2201